src/main/kotlin/app/revanced/patches/youtube/layout/swipecontrols/signatures/SwipeControlBrightnessEnabledParentSignature.kt

```diff
@@ -0,0 +1,19 @@
+package app.revanced.patches.youtube.layout.swipecontrols.signatures
+
+import app.revanced.patcher.annotation.Name
+import app.revanced.patcher.annotation.Version
+import app.revanced.patcher.signature.implementation.method.MethodSignature
+import app.revanced.patcher.signature.implementation.method.annotation.FuzzyPatternScanMethod
+import app.revanced.patcher.signature.implementation.method.annotation.MatchingMethod
+import app.revanced.patches.youtube.layout.swipecontrols.annotations.SwipecontrolsCompatibility
+
+@Name("swipecontrolbrightnessenabled-parent-signature")
+@MatchingMethod(
+    "Ljvy;", "<init>"
```

Comment:
Have you updated this?

Reply:
No, everything is pushed

Reply:
That's not what I meant. It looks like you copy-pasted this from another patch, without updating it accordingly. Could you do so, please?

Reply:
Oh yes now I see what you mean. Will do tomorrow, I'm no longer at home